snorkel/learning/disc_learning.py

```diff
@@ -42,200 +37,85 @@ def score(self, session, X_test, test_labels, gold_candidate_set=None, b=0.5, se
         return s.score(test_marginals, train_marginals, b=b,
```

Comment:
Do we ever set `self.X_train` any more (see lines before, and again comes up below)?  If not we should get rid of this, and get the training marginals in a different way, or just not display them here?